web-app/src/lib/job-input/job-input.ts

```diff
@@ -0,0 +1,84 @@
+import { z } from "zod";
+
+import { JobInputSchemaIntlPath, ValidJobInputTypes } from "./type";
+import { isTypeAndValidationValid } from "./util";
+import { validationSchema } from "./validation";
+
+export const jobInputSchema = (t?: IntlTranslation<JobInputSchemaIntlPath>) =>
+  z
+    .object({
+      id: z.string().nonempty({
+        message: t?.("Id.required"),
+      }),
+      type: z.enum(ValidJobInputTypes, {
+        message: t?.("Type.enum", {
+          options: ValidJobInputTypes.join(", "),
+        }),
+      }),
+      name: z
+        .string()
+        .nonempty({
+          message: t?.("Name.required"),
+        })
+        .min(2, { message: t?.("Name.min") })
+        .max(128, {
+          message: t?.("Name.max"),
+        }),
```

Comment:
I don't know that there are should be any limits. A `string` and `nonempty` check should be enough on our side.

Reply:
@golddydev It should be fixed with 85af789f801e667d5c553a50c3045bc18290455e.

Please double-check and feel free to resolve the discussion if you agree with the change.